python/HHJets.py

```diff
@@ -248,34 +248,33 @@ def __init__(self, df_filter, *args, **kwargs):
 
         self.df_filter = df_filter
 
+        if "/libToolsTools.so" not in ROOT.gSystem.GetLibraries():
```

Comment:
Why this piece of code needs to be outside of the if block starting in `if not os.getenv("_HHJets"):` as it was before?

Reply:
Correction in [e68bb34](https://github.com/jaimeleonh/InferenceTools/pull/4/commits/e68bb3433f748ba48fd6ed12a86bc6258f44c5b9)